pallets/manta-pay/src/types.rs

```diff
@@ -912,11 +912,11 @@ pub struct DensePullResponse {
     /// Ledger Receiver Chunk
     // we decode the receivers/senders with our own way
     #[codec(skip)]
```

Comment:
is this still needed here?
IIRC the  client <-> frontend/sdk connection also uses SCALE encoding, so wouldn't this mean while the client has receivers/senders now, the frontend/sdk will have an empty senders/receivers pair?

Reply:
And I have to remove `TypeInfo`, due to 
```
the trait `TypeInfo` is not implemented for `manta_pay::config::utxo::Checkpoint`
```
`#[codec(skip)]` will solve it, but `Decode/Encode` have to be derived again.

Reply:
cant you just `#[derive(TypeInfo)]` for it?

Reply:
Seems we cannot skip field if only derive `TypeInfo`.
See this issue.